.circleci/config.yml

```diff
@@ -195,9 +197,11 @@ references:
         docker_image_tag=${ECR_ENDPOINT}/${GITHUB_TEAM_NAME_SLUG}/${REPO_NAME}:app-${CIRCLE_SHA1}
         kubectl apply -f kubernetes_deploy/api-sandbox/secrets.yaml
         kubectl set image -f kubernetes_deploy/api-sandbox/deployment.yaml cccd-app=${docker_image_tag} --local -o yaml | kubectl apply -f -
+        kubectl set image -f kubernetes_deploy/cron_jobs/archive_stale.yaml cronjob-worker=${docker_image_tag} --local -o yaml | kubectl apply -f -
         kubectl apply \
         -f kubernetes_deploy/api-sandbox/service.yaml \
-        -f kubernetes_deploy/api-sandbox/ingress.yaml
+        -f kubernetes_deploy/api-sandbox/ingress.yaml \
+        -f kubernetes_deploy/cron_jobs/clean_ecr.yaml
```

Comment:
to be removed, as discussed, because only one namespace needs to do this (because there is only one ECR for `get-paid` and only dev has credentials